restapi/src/main/java/io/stargate/web/resources/v2/schemas/IndexesResource.java

```diff
@@ -0,0 +1,262 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.web.resources.v2.schemas;
+
+import static io.stargate.web.docsapi.resources.RequestToHeadersMapper.getAllHeaders;
+
+import com.codahale.metrics.annotation.Timed;
+import io.stargate.auth.Scope;
+import io.stargate.auth.SourceAPI;
+import io.stargate.auth.UnauthorizedException;
+import io.stargate.db.schema.CollectionIndexingType;
+import io.stargate.db.schema.Column;
+import io.stargate.db.schema.ImmutableCollectionIndexingType;
+import io.stargate.db.schema.Keyspace;
+import io.stargate.db.schema.Table;
+import io.stargate.web.models.Error;
+import io.stargate.web.models.IndexAdd;
+import io.stargate.web.models.IndexKind;
+import io.stargate.web.models.SuccessResponse;
+import io.stargate.web.resources.AuthenticatedDB;
+import io.stargate.web.resources.Db;
+import io.stargate.web.resources.RequestHandler;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import java.util.Map;
+import java.util.Optional;
+import javax.inject.Inject;
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.*;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Api(
+    produces = MediaType.APPLICATION_JSON,
+    consumes = MediaType.APPLICATION_JSON,
+    tags = {"schemas"})
+@Path("/v2/schemas/keyspaces/{keyspaceName}/indexes")
```

Comment:
Since an index belongs to a table I wonder if it would make sense in terms of URI to follow the same pattern as columns

https://github.com/stargate/stargate/blob/master/restapi/src/main/java/io/stargate/web/resources/v2/schemas/ColumnsResource.java#L65

Reply:
+1. 👍